gateway-incoming.yaml

```diff
@@ -219,6 +407,42 @@ components:
             customerID:
               type: string
               description: ID of the customer associated with the subnet containing the IP address.
+    PagedIPResponse:
+      type: object
+      properties:
+        nextPageToken:
+          type: string
+          description: token to be issued in the subsequent request to fetch additional pages. This token should always be the one issued from the previous response
+        results:
+          type: object
```

Comment:
Task: change `results` to type `array` whose `items` are these objects, in order to actually match what is returned. Likewise with `PagedSubnetResponse` below.